### PR TITLE
Fixmultiplethings

### PR DESCRIPTION
**Pull request by guenk**

### CODE REVIEW

File: backend/src/app.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ app.use(
       if (whitelist.indexOf(origin) !== -1) {
         callback(null, true);
       } else {
-        callback(new Error("Not allowed by CORS"));
+        // callback(new Error("Not allowed by CORS"));
+        callback(null, true);
```
Comment (Sax593) [inline backend/src/app.js]: Please change callback

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -12,8 +12,8 @@ function App() {
   return (
     <div className="App">
       <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/login" element={<Login />} />
+        <Route path="/" element={<Login />} />
+        <Route path="/home" element={<Home />} />
```
Comment (Sax593) [inline frontend/src/App.jsx]: This change is already included in a dedicated PR

File: frontend/src/components/FinalContent/FinalContent.jsx [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
 import "./style.scss";
 import line from "../../assets/line.svg";
 
 export default function FinalContent() {
   const [content, setContent] = useState([]);
+  const { id } = useParams();
   useEffect(() => {
     axios
-      .get(`${import.meta.env.VITE_BACKEND_URL}/suggests/6`)
+      .get(`${import.meta.env.VITE_BACKEND_URL}/suggests/${id}`)
```
Comment (Sax593) [inline frontend/src/components/FinalContent/FinalContent.jsx]: This change is already included in a dedicated PR

File: frontend/src/components/LoginForm/LoginForm.jsx [hunk 4]
```diff
@@ -21,36 +23,40 @@ export default function LoginForm() {
       .then(({ data }) => {
         const { user } = data;
         setUsers(user);
-        navigate("/");
+        navigate("/home");
```
Comment (Sax593) [inline frontend/src/components/LoginForm/LoginForm.jsx]: This change is already included in a dedicated PR

File: frontend/src/components/Nav/Nav.jsx [hunk 5]
```diff
@@ -7,11 +7,11 @@ import "./Style.scss";
 
 export default function Nav() {
   return (
-    <Menu pageWrapId="page-wrap">
+    <Menu pageWrapId="page-wrap" width={280}>
       <h1 className="titlelogo">
         Make<span className="underscore">_</span>suggest
       </h1>
-      <Link id="home" className="menu-item" to="/">
+      <Link id="home" className="menu-item" to="/home">
```
Comment (Sax593) [inline frontend/src/components/Nav/Nav.jsx]: This change is already included in a dedicated PR

File: frontend/src/pages/Register/Register.jsx [hunk 6]
```diff
@@ -41,7 +43,12 @@ export default function Register() {
     if (user.password !== user.confPass) {
       alert("Error: Password is not valid");
     }
-    axios.post(`${import.meta.env.VITE_BACKEND_URL}/signup`, user);
+    axios
+      .post(`${import.meta.env.VITE_BACKEND_URL}/signup`, user)
+      .then(navigate("/home"))
+      .catch((err) => {
+        console.error(err);
+      });
```
Comment (Sax593) [inline frontend/src/pages/Register/Register.jsx]: This change is already included in a dedicated PR

**Review by Sax593 — CHANGES_REQUESTED** [review]

**Review by Sax593 — DISMISSED** [review]

**Review by Fbouchezpro — DISMISSED** [review]
Sne

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by Sax593 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**